autoload/neoformat/formatters/haskell.vim

```diff
@@ -5,7 +5,7 @@ endfunction
 function! neoformat#formatters#haskell#hindent() abort
     return {
         \ 'exe' : 'hindent',
-        \ 'args': ['--indent-size.' . shiftwidth(), '--line-length 80'],
+        \ 'args': ['--indent-size. ' . shiftwidth(), '--line-length 80'],
```

Comment:
Should a period in the `--indent-size` arg?

Given a shift width of 2, this would result in an arg of `--indent-size. 2`

When I run

```
cat test.hs | hindent --indent-size. 2 --line-length 80
```

it produces the following error

```
hindent: --indent-size.: openBinaryFile: does not exist (No such file or directory)
```